### ISSUE TITLE
importlib_metadata

### ISSUE THREAD

**Issue by jayvdb** [futursolo/magichttp#6]
In this and magicdict, `importlib_metadata` should be `importlib.metadata` if it is present

```py
try:
    import importlib.metadata as importlib_metadata
except ImportError:
    import importlib_metadata
```

**Comment by futursolo** [comment]
Fixed in https://github.com/futursolo/magichttp/commit/826c95261fb7c74480d75655dea4edebb11143f4.

For `magicdict`, the situation is a little bit more complicated. I never got GitLab CI to work with PyPI after turning on two factor authentication.

I guess the best solution would be to migrate `magicdict` to GitHub and switch to another CI.

I will keep this issue open until I have migrated & updated `magicdict` and updated dependency of this repo.
